Velaptor/Content/Fonts/Font.cs

```diff
@@ -426,4 +397,64 @@ private void RebuildAtlasTexture()
 
         this.metrics = glyphMetrics;
     }
+
+    /// <summary>
+    /// Returns the size of the given single line of <paramref name="line"/>.
```

Comment:
Refactor code docs too

```suggestion
    /// Returns the size of the given single <paramref name="line"/> text.
```